spec/features/bulk_desc_metadata_download_spec.rb

```diff
@@ -11,14 +11,27 @@
 
   it 'New page has a populate druids button and div with last search' do
     visit search_catalog_path q: 'stanford'
-    click_link 'Bulk Actions'
+    within '.search-widgets' do
+      click_link 'Bulk Actions'
+    end
     expect(page).to have_css 'h1', text: 'Bulk Actions'
```

Comment:
this link becomes ambiguous now, and we need to ensure we click the button above search results and not in the top nav